### DISCUSSION TITLE
Wapm.io interactive apps

### DISCUSSION THREAD

**Discussion by DustinBrett** [DustinBrett/daedalOS#184] [Feature Requests]
For example quickjs/qjs. It works with things like `cowsay`.

**Comment by hello-smile6** [comment]
Neofetch doesn't work either.

**Comment by DustinBrett** [comment]
That is not part of wapm.io afaik. Do you know where it is? Can you link it for me please?

**Comment by hello-smile6** [comment]
> That is not part of wapm.io afaik. Do you know where it is? Can you link it for me please?

```
npm i -g neofetch
neofetch
```

**Comment by DustinBrett** [comment]
Yes that is not what I am discussing in this topic.

https://wapm.io/

**Comment by DustinBrett** [comment]
The goal would be something like https://github.com/wasmerio/webassembly.sh but it's become harder with changes to https://github.com/wasmerio/wasmer-js.

**Comment by DustinBrett** [comment]
Wish this project was alive still.
https://github.com/wasmerio/webassembly.sh/issues/79#issuecomment-947896361

**Comment by hello-smile6** [comment]
> Wish this project was alive still. [wasmerio/webassembly.sh#79 (comment)](https://github.com/wasmerio/webassembly.sh/issues/79#issuecomment-947896361)

If it worked still, it'd be nice.

**Comment by DustinBrett** [comment]
Moved to discussions as this isn't a bug. In general I've been waiting/hoping for updates to other repos to make it more clear how I could integrate my xterm into the blocking input system I have seen used. When I have time I will continue the work/discussion here.

**Comment by DustinBrett** [comment]
Looks like eventually wasmer-js will get updated, then I can use it.

https://github.com/wasmerio/wasmer-js/pull/299
